src/presenced.cpp

```diff
@@ -641,6 +643,26 @@ void Command::toString(char* buf, size_t bufsize) const
             snprintf(buf, bufsize, "%s", tmpString.c_str());
             break;
         }
+        case OP_SNSETPEERS:
+        {
+            Id sn = read<uint64_t>(1);
+            uint32_t numPeers = read<uint32_t>(9);
+            string tmpString;
+            tmpString.append("SNSETPEERS - scsn: ");
+            tmpString.append(ID_CSTR(sn));
+            tmpString.append(" num_peers: ");
+            tmpString.append(to_string(numPeers));
+            tmpString.append((numPeers == 1) ? " peer: " :  " peers: ");
```

Comment:
In case of empty-set:
```suggestion
            if (numPeers)
            {
                tmpString.append((numPeers == 1) ? " peer: " :  " peers: ");
            }
```